plugins/time-saver-backend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tduniec/backstage-plugin-time-saver-backend",
-  "version": "0.1.13",
+  "version": "0.1.14",
```

Comment:
This should be Major version patch IMO. This change will break previous functionality, so these are incompatible API changes. We can bump it to 1.0.0

Reply:
Hey @alesgurd and @tduniec. I'm giving you guys a hand and update these myself since I'm too looking forward to see how these changes will affect the latest BS version upgrade on my team.